src/Command/WorkerCommand.php

```diff
@@ -114,6 +119,7 @@ public function execute(Arguments $args, ConsoleIo $io)
     {
         $logger = $this->getLogger($args);
         $processor = new Processor($logger);
+        $processorName = (string)$args->getOption('processor');
```

Comment:
The string casting isn't necessary as the processor name argument can be `null`. Also you can directly use the return value as argument for `bindTo()` instead of assigning to a variable first.

Reply:
@ADmad 
Thanks for the review. I have adjusted it.
psalms pointed out that I needed to cast it explicitly, so I did this.

How about it? If there is a smarter way to write this, please advise. :pray:

Reply:
As psalms pointed out, it is necessary to cast the type to make it explicit, but it is possible to modify it to set the return value directly like this.

```
$client->bindTopic((string)$args->getOption('queue'), $processor, (string)$args->getOption('processor'));
```

The default processor name is set even if the argument processor name is an empty string type.
https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/simple-client/SimpleClient.php#L147

If you need to modify this, please instruct us.

Reply:
Since the `processor` option has not been declared as boolean the return value of `$args->getOption('processor')` will either be a string or `null` only, so that psalm error is not applicable. 

This change is good https://github.com/cakephp/queue/pull/51/commits/52ca45b91103e1ef262e51c97356d851b8566876. Just drop the last commit and add line `/** @psalm-suppress PossiblyInvalidArgument */ above the `$client->bindTopic()` call to suppress the psalm error.

Reply:
@ADmad 
Thank you for being so gracious!
I didn't know about this method. Thank you for your help.
I fixed it.

Reply:
No problem, thank for the PR.